Makefile

```diff
@@ -24,7 +27,18 @@ GMPINCDIR= /usr/include/gmp
 
 # XXXARM: We can't .KEEP_STATE because something confuses everything
 # (directory changes in rules?) and it gets bogus dependencies and always
-# rebuilds everything (which, to be fair, often happens anyway).
+# rebuilds everything.
+
+all:
+	@echo "Targets:"
+	@echo
+	@echo "    download - fetch required sources"
+	@echo "       setup - build all pre-requisites"
+	@echo "     illumos - build illumos"
+	@echo "        disk - build QEMU disk image"
```

Comment:
I'm guessing this is correct here, and changes in the rpi4 bit, right?

Reply:
That was my intention, I forgot to do that bit, fixed now.